TheRocket/TheRocket.ipynb

```diff
@@ -26,59 +229,59 @@
     "discrete_segments = 100\n",
     "\n",
     "# STRUCTURES: General\n",
-    "vehicle_mass_dry = 79 #lb (does not include engine)\n",
-    "vehicle_COM_dry_no_engine = 68 #in\n",
-    "vehicle_diameter = 6 #in\n",
-    "vehicle_height = 151 #in\n",
-    "rail_length = 20 #ft\n",
-    "launch_angle = 5 #degrees\n",
+    "# vehicle_mass_dry = 79 #lb (does not include engine)\n",
```

Comment:
It looks like you are commenting out all of the old declaration which sounds great! Honestly if you have already copied all the code over, you can just delete these lines. It does appear necessary though to remake these variables using the information from your nice excel reading right? Otherwise the cells at the end of the file no longer work if I understand correctly?

Reply:
Yep all the lines i commented out are variables inputted in the excel now so it would have no effect whether they are there or not

Reply:
Sounds good on this front also!